docs/rules/no-async-describe.md

```diff
@@ -0,0 +1,47 @@
+# Disallow async functions passed to describe (no-async-describe)
+
+This rule disallows the use of an async function with `describe`. It usually indicates a copy/paste error or that you're trying to use `describe` for setup code, which should happen in `before` or `beforeEach`. Also, it can lead to [the contained `it` blocks not being picked up](https://github.com/mochajs/mocha/issues/2975).
+
+Example:
+
+```js
+describe(async function () {
```

Comment:
Is this a valid example? Doesn’t `describe` require a string as the first argument?

Reply:
No, it isn't :roll_eyes:

Fixed in 7c9b6dc71664d8756fca64ddbbf0bf7657f727e7